app/views/shared/_image_results.html.erb

```diff
@@ -0,0 +1,7 @@
+<h3>The Images, most recent at the top:</h3>
+<% @images.each do |image| %>
+  <section id='image-<%= image.id%>'>
+    <%= link_to image_tag(image.url), "/images/#{image.id}" %>
+    <p><%= link_to image.keyword.capitalize, "/images/#{image.id}" %></p>
+  </section>
+<% end %>
```

Comment:
Because this was being used in `search index` and `images index`, I moved it to a shared folder